README.md

```diff
@@ -268,16 +270,130 @@ MIT (http://opensource.org/licenses/mit-license.php)
+
+This is how we use the donations:
+
+* Allow the core team to work on weback
+* Thank contributors if they invested a large amount of time in contributing
+* Support projects in the ecosystem that are of great value for users
+* Support projects that are voted most (work in progress)
+* Infrastructure cost
+* Fees for money handling
+
+### Core team
+
+The note shows the main area of involvement. Of course we all contribute in many areas.
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center">
+        <img src="https://github.com/sokra.png?size=150"><br>
+        <a href="https://github.com/sokra">Tobias Koppers</a>
+      </td>
+      <td>
+        Core<br><br>
+        Founder of webpack
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center">
+        <img src="https://github.com/jhnns.png?size=150"><br>
+        <a href="https://github.com/jhnns">Johannes Ewald</a>
+      </td>
+      <td>
+        Styling loaders &amp; plugins<br><br>
+        Early adopter of webpack
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center">
+        <img src="https://github.com/TheLarkInn.png?size=150"><br>
+        <a href="https://github.com/TheLarkInn">Sean T. Larkin</a>
+      </td>
+      <td>
+        Public relations<br><br>
+        Pushed the core team forming
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center">
+        <img src="https://github.com/bebraw.png?size=150"><br>
+        <a href="https://github.com/bebraw">Juho Vepsäläinen</a>
+      </td>
+      <td>
+        Documentation<br><br>
+        Wrote the ebook <a href="https://leanpub.com/survivejs-webpack"><img src="https://cloud.githubusercontent.com/assets/1365881/20286923/93e325c0-aac9-11e6-964d-cabe218c584c.png"> - Webpack</a>
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center">
+        <img src="https://github.com/spacek33z.png?size=150"><br>
+        <a href="https://github.com/spacek33z">Kees Kluskens</a>
+      </td>
+      <td>
+        webpack-dev-server<br><br>
+        <a href="https://codeyellow.nl/"><img height="15" src="https://cloud.githubusercontent.com/assets/1365881/20286583/ad62eb04-aac7-11e6-9c14-a0fef35b9b56.png"></a> supports his open source work on webpack
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+### Early Backers and Sponsors
+
+We had other sources of donations before starting to use OpenCollective. We want to acknowledge these early sponsors and backers, but donations were not public and we are not sure of donors want to stay anonymous. So if you want to be in this list, just drop @sokra a note via mail.
```

Comment:
just drop @sokra a note via mail. -> just drop [@sokra](https://github.com/sokra) a note via mail.
I think it would be better to link here.

Seems not to be a link in GitHub.
https://github.com/webpack/webpack/blob/60ea7008e3f80523c1a2acbb945138a98fa1dbb9/README.md#early-backers-and-sponsors